triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Runner.scala

```diff
@@ -468,10 +465,12 @@ class Runner(
           logger.error(s"Unexpected exception: $e")
       })
     }
-    source
-      .viaMat(msgFlow)(Keep.right[NotUsed, T])
-      .toMat(getTriggerSink(name, acs, submit))(Keep.both)
-      .run()
+    val (t, futureValue) =
+      source
+        .viaMat(msgFlow)(Keep.right[NotUsed, T])
+        .toMat(getTriggerSink(name, acs, submit))(Keep.both)
+        .run()
+    (t, futureValue.map(SEValue(_)))
```

Comment:
If we always return an `SValue` can we just change the return type? It doesn’t seem useful to wrap it here.

Reply:
There's lots of callers in the test which would need adapting. Happy to do that if that's what you prefer.

Reply:
Let’s do that. Should hopefully be easy enough to change.

Reply:
Let's do it but in a separate PR.

Reply:
Done. Turned out most callers funnelled through a single point `toResult` where the change could be made.
Thanks.